src/main/java/org/lwes/emitter/MulticastEventEmitter.java

```diff
@@ -145,60 +133,10 @@ public int getTimeToLive() {
   }
 
   /**
-   * Sets the ESF file used for event validation.
-   *
-   * @param esfFilePath the path of the ESF file
-   */
-  public void setESFFilePath(String esfFilePath) {
-    if (getFactory() != null) {
-      getFactory().setESFFilePath(esfFilePath);
-    }
-  }
-
-  /**
-   * Gets the ESF file used for event validation
-   *
-   * @return the ESF file path
-   */
-  public String getESFFilePath() {
-    if (getFactory() != null) {
-      return getFactory().getESFFilePath();
-    }
-    else {
-      return null;
-    }
-  }
-
-  /**
-   * Sets an InputStream to be used for event validation.
-   *
-   * @param esfInputStream an InputStream used for event validation
-   */
-  public void setESFInputStream(InputStream esfInputStream) {
-    if (getFactory() != null) {
-      getFactory().setESFInputStream(esfInputStream);
-    }
-  }
-
-  /**
-   * Gets the InputStream being used for event validation.
-   *
-   * @return the InputStream of the ESF validator
-   */
-  public InputStream getESFInputStream() {
-    if (getFactory() != null) {
-      return getFactory().getESFInputStream();
-    }
-    else {
-      return null;
-    }
-  }
-
-  /**
-   * Initializes the emitter.
+   * Creates the multicast <code>MulticastSocket</code>.
    */
   @Override
-  public void initialize() throws IOException {
+  protected void createSocket() throws IOException {
```

Comment:
is the API still backward compatible? If not, should the version change not be a major bump?